src/python.rs

```diff
@@ -943,6 +943,7 @@ fn deploy_evm(
     sol_code_path: PathBuf,
     rpc_url: Option<String>,
     optimizer_runs: usize,
+    private_key: Option<String>,
```

Comment:
can you explicitly set the default on the pyOxide binding macro above

Reply:
you also need to do this for deploying the da verifier in the python bindings

Reply:
Thanks for the review, I'll take care of those bindings. Just be patient, I think I can do it during the weekend, and I'll push another commit with the updates.

Reply:
awesome ty :)

Reply:
Done. I also took the opportunity to add some additional tests in `integration_tests.rs` and in `binding_tests.py` for this feature.